### PR TITLE
Implemented knock back enchantment.

### PR DESCRIPTION
**Pull request by JustTalDevelops**

### CODE REVIEW

File: server/player/player.go [hunk 1]
```diff
@@ -1446,6 +1447,12 @@ func (p *Player) AttackEntity(e world.Entity) {
 		}
 		if vulnerable {
 			p.Exhaust(0.1)
+
+			if s, ok := i.Enchantment(enchantment.KnockBack{}); ok {
+				diff := float64(s.Level()) / 2
+				force += diff
+				height += diff
+			}
```
Comment (Sandertv) [inline server/player/player.go:1455]: Would maybe move these calculations into a method the enchantment itself.

Comment (JustTalDevelops) [reply server/player/player.go]: Changed.

**Review by Sandertv — COMMENTED** [review]
Is this tested to match vanilla?

**Comment by JustTalDevelops** [comment]
Yes.

**Pull request merged**